en_US/getting-started/config.md

```diff
@@ -42,7 +42,7 @@ Usually we will list all its optional values near the configuration item of type
 
 **ip**
 
-When you see that the data type of a configuration item is `ip`, it means that you can set the configuration item in the form of <IP>: <Port>`, for example, `0.0.0.0: 1883`.
```

Comment:
we try to avoid using `<text>` for placeholders in docs.
please help to change to `{ip}:{port}` instead.

Reply:
@zmstone  Do you mean,  `<ip>:<port>` should be replaced with `{ip}:{port}`?

```diff
- When you see that the data type of a configuration item is `ip`, it means that you can set the configuration item in the form of `<ip>:<port>`, for example, `0.0.0.0:1883`.
+ When you see that the data type of a configuration item is `ip`, it means that you can set the configuration item in the form of `{ip}:{port}`, for example, `0.0.0.0:1883`.
```

Reply:
yes please.
This is because the md to html rendering engine may treat <foobar> like an html tag. (Although this one seemingly did cause any trouble)